### PR TITLE
frontend/dashboard-doctor-table

### PR DESCRIPTION
**Pull request by Smokey4869-crypto**
## Motivation and Context

The on-demand telehealth platform must be able to dynamically display a list of available doctors on the Patient dashboard to allow patients to find suitable doctors based on different search criteria and request a video consultation with their preferred doctor.

**JIRA issue:** https://swe40001-group-42.atlassian.net/browse/SEPAB-

## Changed

1. Fetch all available doctors on Doctor Table

## How Has This Been Tested?

TODO: Unit tests for Doctor table

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

File: frontend/src/components/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -59,6 +59,8 @@ const Dashboard = () => {
     surname: Userfront.user.data.surName,
   };
 
+  console.log(Userfront.user.data.givenName)
```
Comment (EddieeeM) [inline frontend/src/components/Dashboard/Dashboard.tsx]: Please remove/comment this out once you are done with it

**Review by EddieeeM — CHANGES_REQUESTED** [review]
I noticed **_`userId`_**  is added in all models, did you check if the Axios calls still pass and update correctly in the MongoDB database?

File: frontend/src/components/Dashboard/Dashboard.tsx (hunk 1 above)
Comment (Smokey4869-crypto) [reply frontend/src/components/Dashboard/Dashboard.tsx]: I have removed/commentted all unnecessary codes

**Review by EddieeeM — APPROVED** [review]
Looks good to merge, thanks!

**Pull request merged**